src/api/forms/service.js

```diff
@@ -189,6 +189,55 @@ export async function updateDraftFormDefinition(formId, definition, author) {
   }
 }
 
+/**
+ * @param {string} formId - ID of the form
+ * @param {Partial<FormMetadataInput>} formUpdate - full JSON form definition
+ * @returns {Promise<string>}
+ */
+export async function updateFormMetadata(formId, formUpdate) {
+  logger.info(`Updating form metadata for form ID ${formId}`)
+
+  try {
+    // Get the form metadata from the db
+    const form = await getForm(formId)
+
+    if (form.live && formUpdate.title) {
```

Comment:
Can I suggest a tweak to this too?

Otherwise you could set `{ title: '' }` (falsy value) and it wouldn't be caught by this check

```suggestion
    if (form.live && 'title' in formUpdate) {
```